src/charts/bar.js

```diff
@@ -698,10 +742,25 @@ define(function(require) {
 
         // API
 
+        /**
+         * Gets or Sets the gradient colors of a bar in the chart
+         * @param  {String[]} _x Desired color gradient for the line (array of two hexadecimal numbers)
+         * @return {String[] | module} Current color gradient or Line Chart module to chain calls
+         * @public
+         */
+        exports.barGradient = function(_x) {
```

Comment:
Maybe we could call this 'chartGradient', as the gradients are not being applied to each bar, but to the whole chart